src/Text/Pandoc/Writers/OpenDocument.hs

```diff
@@ -148,6 +151,11 @@ withTextStyle s f = do
   modify $ \st -> st{ stTextStyleAttr = oldTextStyleAttr }
   return res
 
+inSourceCodeTags :: PandocMonad m => Doc Text -> OD m (Doc Text)
+inSourceCodeTags d = do
+  modify $ \st -> st { stFirstPara = False }
```

Comment:
Does the paragraph style you're using for this ensure that the first line is not indented?
(That was the main point of setting stFirstPara = True, so I just wanted to be sure.)

Reply:
I intended to disable `stFirstPara` but sorry this is my I misunderstanding. This Line (L156) will be removed in next commit.
Followings are memo for future who are reading this commit (maybe for me).

I saw `CodeBlock` calls [`Para` ](https://github.com/jgm/pandoc/blob/master/src/Text/Pandoc/Writers/Docx.hs#L985) and `Para` disables [`stFirstPara`](https://github.com/jgm/pandoc/blob/master/src/Text/Pandoc/Writers/Docx.hs#L970) .
But again, `CodeBlock` enables [`stFirstPara`](https://github.com/jgm/pandoc/blob/master/src/Text/Pandoc/Writers/Docx.hs#L986) . So, I will change the code as following.

```
    | CodeBlock    a s <- bs = setFirstPara >> (inlinesToOpenDocument o [Code a s] >>= inSourceCodeTags)
```

Basically, I would like to change the odt output to be the same as or similar to docx.